pkgs/servers/windmill/ui.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildNpmPackage
+}:
+
+buildNpmPackage rec {
+
+  pname = "windmill-ui";
+  version = "1.56.1";
+
+  src = fetchFromGitHub {
+    owner = "windmill-labs";
+    repo = "windmill";
+    rev = "v${version}";
+    sha256 = "sha256-M2/tTZFV2YjrykLVELjFE9iQhzcrQtRCIXZ2EpTpuE8=";
+  };
+
+  sourceRoot = "source/frontend";
+
+  npmDepsHash = "sha256-884zjHhiuFXr+Z4a/AcEksGbakRcghromQ5QQj0qBAY=";
+
+  preBuild = ''
+    npm run generate-backend-client
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share
+    mv build $out/share/windmill-frontend
+  '';
```

Comment:
Missing meta

Reply:
Added meta, but did not make sense not to use the package name in the description. Hope that is alright, since this is not a top-level package.